nunaserver/pyproject.toml

```
@@ -0,0 +1,9 @@
+# --------------------------------------------------  BLACK  --------------------------------------------------
+[tool.black]
+line-length = 120
+target-version = ['py311']
+include = '''
+(src/.*\.pyi?$)
```

```suggestion
(nunaserver/.*\.pyi?$)
```

Please reformat the code afterward using Black.